.github/workflows/build-and-test-inplace.yml

```diff
@@ -19,11 +19,12 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install pytest setuptools numpy pyzmq pillow PyQt5 pyqtgraph matplotlib
+          pip install ".[dev]"
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
       - name: Build C extension
         run: |
           make
       - name: Run tests using pytest
         run: |
-          make test
+          make test_gh
```

Comment:
Is it possible to avoid having a gh specific version?

Reply:
now when we separate format checks from it is not needed